app/controllers/spree/admin/products_controller_decorator.rb

```diff
@@ -92,9 +94,28 @@ def collection_actions
 
   private
 
+  def latest_import
+    @show_latest_import = params[:latest_import] || false
+  end
```

Comment:
what is the benefit of having it as a before_filter instead of an explicit method call, in this case? you already know my preference :trollface: